web_widget_float_highlight/static/src/js/web_highlight.js

```diff
@@ -0,0 +1,66 @@
+odoo.define("web.float.highlight", function (require) {
+    "use strict";
+    var form_widgets = require("web.form_widgets");
+
+    form_widgets.FieldFloat.include({
+
+        init: function () {
+            this._super.apply(this, arguments);
+
+            var load_defaults = (this.options && this.options.load_defaults) || false;
+
+            var options = {
+                lower_threshold: (this.options && this.options.lower_threshold) || 0,
+                upper_threshold: (this.options && this.options.upper_threshold) || 0,
+                lower_bg_color: (this.options && this.options.lower_bg_color) || load_defaults && "red" || "white",
+                middle_bg_color: (this.options && this.options.middle_bg_color) || "white",
+                upper_bg_color: (this.options && this.options.upper_bg_color) || load_defaults && "green" || "white",
+                lower_font_color: (this.options && this.options.lower_font_color) || load_defaults && "white" || "#666666",
+                middle_font_color: (this.options && this.options.middle_font_color) || "#666666",
+                upper_font_color: (this.options && this.options.upper_font_color) || load_defaults && "white" || "#666666",
```

Comment:
Can't you use bootstrap status classes instead?
http://getbootstrap.com/css/#helper-classes-colors